bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF

```diff
@@ -9,6 +9,7 @@ Fragment-Host: org.eclipse.smarthome.core
 Import-Package: 
  groovy.lang,
  javax.measure.quantity,
+ net.bytebuddy,

```

Comment:
Is this still necessary (WRT #5319)? I also don't identify the place you are using that package.